tests/upgrade_operator/upgrade_operator_suite_test.go

```diff
@@ -46,16 +46,13 @@ func TestLifecycle(t *testing.T) {
 func InstallOldOperator() {
 	step := "install old Cass Operator v1.1.0"
 	By(step)
-	err := helm_util.Install(oldOperatorChart, "cass-operator", ns.Namespace, map[string]string{})
-	mageutil.PanicOnError(err)
+	ginkgo_util.HelmInstallWithOverrides(oldOperatorChart, ns.Namespace, map[string]string{})
 }
 
 func UpgradeOperator() {
 	step := "upgrade Cass Operator"
 	By(step)
-	var overrides = map[string]string{"image": cfgutil.GetOperatorImage()}
-	err := helm_util.Upgrade("../../charts/cass-operator-chart", "cass-operator", ns.Namespace, overrides)
-	mageutil.PanicOnError(err)
+	ginkgo_util.HelmUpgradeWithOverrides("../../charts/cass-operator-chart", ns.Namespace, map[string]string{})
```

Comment:
For the sake of symmetry with `InstallOldOperator`, I would leave this function unchanged. This test is testing the behavior of changing the operator image, so we want to let it be explicit about what it is doing.